tests/test_flows.py

```diff
@@ -112,6 +113,14 @@ def tcpdump_start(self):
             else:
                 self.err("tcpdump terminated")
                 self.tcpdump_proc = None
+            # after tcpdump finishes, add a list of all ieee1905 packets for easier analysis
+            # first, generate a json file of the pcap dump
+            self.json_from_pcap = subprocess.run(['tshark', '-r', outputfile, '-T', 'json' '>tmp.json'])
```

Comment:
Ugh... How about just parsing the pcap file with [pypcap](https://pypi.org/project/pcap-ct/) instead of relying on an external tool?

Reply:
I've played around with a bunch of python pcap libraries and none allow for convenient work with ieee1905 messages. on the other hand the objects generated using this method are very readable and convenient.
I don't believe I've checked pypcap, I've tried 2 other libraries, dpkg and scapy.
I'll check it out right now.